content/docs/reactivesearch/searchbox/Quickstart.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'QuickStart'
+meta_title: 'QuickStart to SearchBox'
+meta_description: 'SearchBox is a lightweight search library with some common utilities.'
+keywords:
+    - quickstart
+    - react-searchbox
```

Comment:
This should be searchbox.
And also I think you forgot to add `Link` in `NavBar.js`

Reply:
Thanks good catch 🥇